packages/editor/src/utils/terms.js

```diff
@@ -41,7 +46,7 @@ export function buildTermsTree( flatTerms ) {
 
 // Lodash unescape function handles &#39; but not &#039; which may be return in some API requests.
```

Comment:
We can probably remove this comment now.

Reply:
Good call 👍 Removed.